tool/update_version.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2019 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+VERSION=$1
+LAST_VERSION="$(sed -n -e 's/^.*version: //p' packages/devtools_app/pubspec.yaml)"
+
+echo "The previous version was: $LAST_VERSION"
+
+if [ -z "$VERSION" ]; then
+    echo "No version specified."
+    echo "Usage: tool/update_version.sh NEW_VERSION_NUMBER"
+    exit 1
+fi
+
+# If you add a package that is version locked, please add it to this list.
+PUBSPECS="./devtools_shared/pubspec.yaml ./devtools_server/pubspec.yaml ./devtools/pubspec.yaml ./devtools_app/pubspec.yaml ./devtools_testing/pubspec.yaml"
```

Comment:
Minor nit would this be a little more findable?

PUBSPECS="./devtools_shared/pubspec.yaml "\
"./devtools_server/pubspec.yaml "\
"./devtools/pubspec.yaml "\
"./devtools_app/pubspec.yaml "\
"./devtools_testing/pubspec.yaml"

Reply:
good idea. done.